### DISCUSSION TITLE
Stop autoscaling based on custom metric

### DISCUSSION THREAD

**Discussion by spereverziev** [kedacore/keda#5161] [Q&A / Need Help]
Is it possible to pause autoscaling based on custom metric like `rps` ? I see that there is `autoscaling.keda.sh/paused` but I want it to be set automatically. My use case is that when there is a upstream outage, I don't want to scale down the service.

**Comment by zroubalik** [comment]
Hi, at the moment it is not possible. Feel free to open an issue with the proposal.
